src/modules/SX127x/SX1278.cpp

```diff
@@ -342,6 +335,29 @@ int16_t SX1278::setOutputPower(int8_t power, bool useRfo) {
   return(state);
 }
 
+int16_t SX1278::checkOutputPower(int8_t power, int8_t* clipped, bool useRfo) {
+  // check allowed power range
+  if(useRfo) {
+    // RFO output
+    if(clipped) {
+      *clipped = RADIOLIB_MAX(-3, RADIOLIB_MIN(15, power));
+    }
+    RADIOLIB_CHECK_RANGE(power, -3, 15, RADIOLIB_ERR_INVALID_OUTPUT_POWER);
+  } else {
+    // PA_BOOST output, check high-power operation
+    if(clipped) {
+      if(power == 20) {
+        *clipped = 20;
+      }
+      *clipped = RADIOLIB_MAX(2, RADIOLIB_MIN(17, power));
```

Comment:
Cppcheck (rightfully) complains about this one: https://github.com/jgromes/RadioLib/actions/runs/8884715530/job/24394437946?pr=1080#step:4:2427